src/EA.Weee.Web.Tests.Unit/Areas/Aatf/Mapping/ToViewModel/ViewEvidenceNoteViewModelMapTests.cs

```diff
@@ -625,5 +629,36 @@ public void Map_GivenSourceWhereNoteStatusIsDraftOrReturnedAndAatfCannotEditEvid
             //assert
             result.DisplayEditButton.Should().BeFalse();
         }
+
+        [Fact]
+        public void Map_GivenSourceWithHistoryNoteData_EvidenceNoteHistoryViewModelShouldBePopulated()
+        {
+            //arrange
+            var source = TestFixture.Create<ViewEvidenceNoteMapTransfer>();
+            var data = new EvidenceNoteHistoryData(TestFixture.Create<Guid>(), TestFixture.Create<NoteStatus>(), TestFixture.Create<int>(), TestFixture.Create<NoteType>(), TestFixture.Create<DateTime?>(), TestFixture.Create<string>());
+            var history = new List<EvidenceNoteHistoryData>()
+            {
+                data,
+            };
+            source.EvidenceNoteData.EvidenceNoteHistoryData = history;
+
+            A.CallTo(() => mapper.Map<IList<EvidenceNoteHistoryViewModel>>(A<EvidenceNoteHistoryDataViewModelMapTransfer>._)).Returns(new List<EvidenceNoteHistoryViewModel>()
+            {
+                new EvidenceNoteHistoryViewModel(data.Id, data.Reference, data.TransferredTo, data.Type, data.Status, data.SubmittedDate),
+            });
+
+            //act
+            var result = map.Map(source);
+
+            //assert
+            result.EvidenceNoteHistoryData.First().Id.Should().Be(history.First().Id);
+            result.EvidenceNoteHistoryData.First().Status.Should().Be(history.First().Status);
+            result.EvidenceNoteHistoryData.First().Reference.Should().Be(history.First().Reference);
+            result.EvidenceNoteHistoryData.First().Type.Should().Be(history.First().Type);
+            result.EvidenceNoteHistoryData.First().SubmittedDate.Should().Be(history.First().SubmittedDate);
+            result.EvidenceNoteHistoryData.First().TransferredTo.Should().Be(history.First().TransferredTo);
+
+            A.CallTo(() => mapper.Map<IList<EvidenceNoteHistoryViewModel>>(A<EvidenceNoteHistoryDataViewModelMapTransfer>._)).MustHaveHappenedOnceExactly();
```

Comment:
rather than doing ._  here do a Sequence equal check to ensure the list passed into the mapper is as expected